lib/api.js

```diff
@@ -0,0 +1,124 @@
+var base      = require('taskcluster-base');
```

Comment:
This is the lamest review comment ever, but .. shouldn't this and every file have an MPL header?